js-challenge/bbs-api/users.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8" />
+    <title>ユーザー編集</title>
+</head>
+
+<body>
+    <form id="fetchUpdate">
+        <input type="text" name="bio" id="updateBio" placeholder="自己紹介">
+        <input type="button" value="更新" id="upgradeSubmit" class="upBtn">
+        <input type="button" value="削除" id="destoroySubmit" class="desBtn">
```

Comment:
```suggestion
        <input type="button" value="削除" id="destroySubmit" class="desBtn">
```
綴り間違いは検索のときに困るので直しましょう。